src/ui/inspector/Inspector.vue

```diff
@@ -35,30 +81,60 @@ export default {
         Elements,
         Properties,
         Location,
-        InspectorViews
+        InspectorViews,
+        TabbedInspectorView
     },
     props: {
         'isEditing': Boolean
     },
     data() {
         return {
-            hasComposition: false
+            hasComposition: false,
+            hasTabs: false,
+            tabbedViews: [],
+            currentTabbedView: {},
+            PROPERTY_VIEW: {
+                key: '__properties',
+                getName: () => { return 'Properties';}
+            }
         }
     },
     mounted() {
-        this.openmct.selection.on('change', this.refreshComposition);
-        this.refreshComposition(this.openmct.selection.get());
+        this.openmct.selection.on('change', this.updateInspectorViews);
     },
     destroyed() {
-        this.openmct.selection.off('change', this.refreshComposition);
+        this.openmct.selection.off('change', this.updateInspectorViews);
     },
     methods: {
+        updateInspectorViews(selection) {
+            this.refreshComposition(selection);
+            this.refreshTabs(selection);
+        },
         refreshComposition(selection) {
             if (selection.length > 0 && selection[0].length > 0) {
                 let parentObject = selection[0][0].context.item;
 
                 this.hasComposition = !!(parentObject && this.openmct.composition.get(parentObject));
             }
+        },
+        refreshTabs(selection) {
+            if (selection.length > 0 && selection[0].length > 0) {
+                let selectedViews = this.openmct.inspectorViews.get(selection);
+                this.tabbedViews = selectedViews.filter(selectedView => {
+                    return typeof selectedView.tabbed === 'function' && selectedView.tabbed();
+                });
+                this.hasTabs = this.tabbedViews.length;
```

Comment:
Computed properties are perfect for this sort of thing.

Reply:
Could you elaborate? which property would be computed here?